packages/p2p-v2/src/components/Input/Input.scss

```diff
@@ -0,0 +1,50 @@
+.p2p-v2-input {
+    margin: 0.5rem 0 3.6rem;
+
+    @include mobile {
+        margin-bottom: 1rem;
+    }
+
+    &__error {
+        padding-top: 0.2rem;
+        position: absolute;
+        margin-left: 1.6rem;
+
+        @include mobile {
+            box-decoration-break: clone;
+            padding-top: 0.4rem;
+            margin-bottom: 0.6rem;
+            position: relative;
+            width: 296px;
+        }
+    }
+
+    &__field {
+        display: flex;
+        width: 360px;
+        height: 40px;
+        padding: 10px 16px;
```

Comment:
should we have the styles in rem or px?

Reply:
yeah, better we follow rem, i was following the design sizing